src/xdocs/config_coding.xml

```diff
@@ -4953,6 +4953,73 @@ if (&quot;something&quot;.equals(x))
       </subsection>
     </section>
 
+    <section name="UnnecessarySemicolonAfterTypeMemberDeclaration">
+      <p>Since Checkstyle 8.24</p>
+      <subsection name="Description"
+                  id="UnnecessarySemicolonAfterTypeMemberDeclaration_Description">
+        <p>
+          Checks if unnecessary semicolon is used after code type member declaration.
```

Comment:
There is nothing more to add to the description on why user should care to remove any unnecessary semicolons?

Reply:
What do you suggest?

Reply:
Maybe a reason why someone should care about removing unnecessary semicolons. Where it could create some type of misunderstanding.

Reply:
Can you give an example? Because I cannot imagine something that is not obvious.

Reply:
other Checks for unnecessary semicolon:
https://checkstyle.org/config_coding.html#UnnecessarySemicolonInEnumeration
https://checkstyle.org/config_coding.html#UnnecessarySemicolonInTryWithResources
for Parentheses:
https://checkstyle.org/config_coding.html#UnnecessaryParentheses

We do not provide any further details on reason or rationale. It is matter of style, somebody like such style some do not.
Some people think it is more distinctive to have extra characters. There same is for modifiers on methods,  some people like explicit modifiers some try to minimize characters in code by all means (removal of extra/default modifiers).
I do NOT know, how to improve brief description.
@rnveach , please suggest exact wording you would like to have there.

>Checks if unnecessary semicolon is used after code type member declaration.

let make it  `Checks if unnecessary semicolon is used after type member declaration.`.
I am not not sure what `code` mean.

Reply:
It just feels empty that we don't add more.
Besides @romani's suggestion, I was thinking along the same thoughts as what you guys were saying. Extra semi-colons don't after type declaration can't cause any issues or mis-understanding with the code. It is just unnecessary and can be removed if user doesn't want them in.

Reply:
we are not targeting `;` inside methods by this Check.

Reply:
It was just an example on how other unnecessary has a more important reasoning than this one.

Reply:
Are we not doing for this item?

Reply:
As I said before, I don't think this check requires additional explanations.